core/domain/feedback_domain.py

```diff
@@ -19,11 +19,30 @@
 
 
 class FeedbackThread(object):
-    """Domain object for a feedback thread."""
+    """Domain object for a feedback thread.
+    
+    Attributes:
+        full_thread_id: str. The feedback thread ID.
+        exploration_id: str. The associated exploration's ID.
+        state_name: str. The name of the state associated with 
+            the feedback thread.
+        original_author_id: str. The ID of the original author.
+        status: string(select). The current status of the 
+            thread.
+        subject: str. The subject of the feedback thread.
+        summary: str. A summary of the feedback thread.
+        has_suggestion: bool. Indicates whether a given exploration has a 
+            change suggestion.
```

Comment:
The current docstring is incorrect. This is a boolean attribute that refers to whether the thread is a 'suggestion thread' or not.

So I would suggest: "bool. Whether the feedback thread includes a suggestion."